src/btree/bt_debug.c

```diff
@@ -1256,7 +1263,8 @@ __debug_cell(WT_DBG *ds, const WT_PAGE_HEADER *dsk, WT_CELL_UNPACK *unpack)
     WT_DECL_ITEM(buf);
     WT_DECL_RET;
     WT_SESSION_IMPL *session;
-    char ts_string[3][WT_TS_INT_STRING_SIZE];
+    char tp_string[2][WT_TP_STRING_SIZE];
+    char ts_string[WT_TS_INT_STRING_SIZE];
```

Comment:
This could be `char x, y;`.

Reply:
There are other uses of this naming throughout the code. Shall I simplify them all?